nautobot_netbox_importer/tests/test_import.py

```diff
@@ -265,3 +218,25 @@ def _generate_fixtures(wrapper: NautobotModelWrapper, output_path: Path):
         pks=",".join(str(instance.pk) for instance in random_instances),
         output=output_path,
     )
+
+
+def _create_test_cases():
+    """Dynamically create test cases, based on the current Nautobot version and defined fixtures."""
+    nautobot_version_str = ".".join(nautobot_version.split(".")[:2])
+    fixtures_path = Path(__file__).parent / "fixtures" / f"nautobot-v{nautobot_version_str}"
+    if not fixtures_path.is_dir() or not (fixtures_path / "dump.sql").is_file():
+        raise ValueError(f"Fixtures for Nautobot {nautobot_version_str} are missing")
+
+    def add_case(fixtures_name: str):
+        def test_method(self):
+            # pylint: disable=protected-access
+            self._import(fixtures_name, fixtures_path / fixtures_name)
+
```

Comment:
# Creates dynamic test methods, ex: TestImport.test_3_7_custom

Reply:
Added more documentation related to test fixtures.